Sources/NIOExtras/LengthFieldBasedFrameDecoder.swift

```diff
@@ -13,6 +13,38 @@
 //===----------------------------------------------------------------------===//
 
 import NIO
+import Foundation
```

Comment:
Foundation import

Reply:
Good catch! All Foundation imports are leftovers from the initial generic implementation which used `Data` & `DataProtocol`.